PROTOCOL.md

```diff
@@ -155,6 +156,46 @@ to the stack before jumping to the kernel.
 
 All other general purpose registers are set to 0.
 
+### aarch64
+
+`PC` will be the entry point as defined as part of the executable file format,
+unless the an Entry Point feature is requested (see below), in which case,
+the value of `PC` is going to be taken from there.
+
+The contents of the `VBAR_EL1` register are undefined, and the kernel must load it's own.
```

Comment:
Nitpick, the kernel must load it's own is wrong, the kernel must load its own is better. We don't mean load it is own here.